src/commands/list.ts

```diff
@@ -7,15 +7,20 @@ import { leetCodeManager } from "../leetCodeManager";
 import { ProblemState, UserStatus } from "../shared";
 import { DialogType, promptForOpenOutputChannel } from "../utils/uiUtils";
 
-export interface IProblem {
-    favorite: boolean;
-    locked: boolean;
-    state: ProblemState;
-    id: string;
-    name: string;
-    difficulty: string;
-    passRate: string;
-}
+// tslint:disable-next-line:typedef
+export const IProblemDefault = {
+    favorite: false,
+    locked: false,
+    state: ProblemState.Unknown,
+    id: "",
+    name: "",
+    difficulty: "",
+    passRate: "",
+    companies: [] as string[],
+    tags: [] as string[],
+};
+
+export type IProblem = typeof IProblemDefault;
```

Comment:
It would be better to put the `IProblem` definition and `IProblemDefault` to other places instead of in this file.
Let's put them into `shared.ts`, I'll refactor this part later.